mdhhockey/constants.py

```diff
@@ -1,12 +1,37 @@
+import configparser
+
+# Config File (for vars that shouldn't be uploaded to GitHub)
+config = configparser.ConfigParser()
+config.read(['/home/jeremy/mdh-hockey/mdhhockey/config.cfg'])
```

Comment:
Could use a local path, but honestly not really a big deal, and is quite obvious to change once it errors out.

Reply:
Local path doesn't work once I start running it via cron, because the working directory it's invoked from is not the same. At least for my OTH stuff I've found that just using absolute paths works best, but I'm open to suggestions here.